include/onnxruntime/core/graph/graph.h

```diff
@@ -101,7 +101,7 @@ class Node {
   Node::Type NodeType() const noexcept;
 
   /** Gets the function body if the #NodeType is fused, or nullptr if not. */
-  const Function* GetFunctionBody() const noexcept;
+  Function* GetFunctionBody() noexcept;
```

Comment:
Please keep the const variant and maybe call the non-const one GetMutableFunctionBody for consistency with other methods here.